api/v1/controllers/EcosystemController.js

```diff
@@ -0,0 +1,76 @@
+var _ = require('lodash');
+var bodyParser = require('body-parser');
+var middleware = require('../middleware');
+var router = require('express').Router();
+var _Promise = require('bluebird');
+
+var errors = require('../errors');
+var config = require('../../config');
+var requests = require('../requests');
+var roles = require('../utils/roles');
+
+var EcosystemService = require('../services/EcosystemService');
+
+
+function createEcosystem (req, res, next) {
+	var name = req.body.name;
+
+	EcosystemService
+		.createEcosystem(name)
+		.then(function (newEcosystem) {
+			res.body = {};
+			res.body.ecosystem = newEcosystem.attributes.name;
```

Comment:
Since the `body` represents an `Ecosystem`, it should have all of the attributes. Just do `res.body = newEcosystem.toJSON()` and you should be set.